iocage/cli/create.py

```diff
@@ -68,93 +73,95 @@ def validate_count(ctx, param, value):
               help="Provide a specific UUID for this jail.")
 @click.option("--basejail", "-b", is_flag=True, default=False,
               help="Set the new jail type to a basejail. Basejails"
-                   " mount the specified RELEASE directories as nullfs"
-                   " mounts over the jail's directories.")
+                   " mount the specified RELEASE directories"
+                   " over the jail's directories.")
+@click.option("--basejail-type", type=click.Choice(['nullfs', 'zfs']),
+              help="The method of mounting release datasets into"
+                   " the basejail on start.")
 @click.option("--empty", "-e", is_flag=True, default=False,
               help="Create an empty jail used for unsupported or custom"
                    " jails.")
 @click.option("--short", "-s", is_flag=True, default=False,
               help="Use a short UUID of 8 characters instead of the default"
                    " 36.")
+@click.option("--no-fetch", is_flag=True, default=False,
+              help="Do not automatically fetch releases")
 @click.option("--force", "-f", is_flag=True, default=False,
               help="Skip the interactive question.")
+@click.option("--log-level", "-d", default="info")
 @click.argument("props", nargs=-1)
-def cli(release, template, count, props, pkglist, basejail, empty, short,
-        name, _uuid, force):
+def cli(release, template, count, props, pkglist, basejail, basejail_type,
+        empty, short, name, _uuid, no_fetch, force, log_level):
+
+    if short is True:
```

Comment:
@We should have constant style. Either we use the more explicit `is True`, or we stick with the `if foo`(my preference).